### ISSUE TITLE
bug: restoring partitioned tables fails

### ISSUE THREAD

**Issue by janmeier** [GreenmaskIO/greenmask#238]
Given the following configuration file, and the schema from the partitoned table example https://docs.greenmask.io/latest/built_in_transformers/transformation_inheritance/?h=partitioned#example-1-partitioned-tables
```
common:
  tmp_dir: '/tmp'

log:
  level: debug

storage:
  directory:
    path: /tmp/dumps

dump:
  pg_dump_options:
    load-via-partition-root: true

  transformation:
    - schema: public
      name: sales
      apply_for_inherited: true
      transformers:
        - name: RandomDate
          params:
            min: "2023-01-01"
            max: "2023-03-01"
            column: "sale_date"
            engine: "random"
```

`greenmask restore` fails with
```
> fatal error="data stage restoration error: at least one worker exited with error: unable to perform restoration task (worker 1 restoring table \"public\".\"sales\"): cannot open dump file: open /tmp/dumps/1731395980666/3400.dat.gz: no such file or directory" pid=14714
```

It seems the dump only contains data for the partitions, not for the root table.

This is not related to the `apply_for_inherited` configuration option, because the error happens even if I don't apply any transformations to the table.

This regression seems to have appeared in `v0.2.1`

**Comment by wwoytenko** [comment]
Thank you for reporting. This will be fixed by #241

**Comment by wwoytenko** [comment]
@janmeier Could you please check if these changes resolve the issue?

**Comment by wwoytenko** [comment]
Published in v0.2.3
